### DISCUSSION TITLE
电信CN2家宽已经被屏蔽所有中转机场

### DISCUSSION THREAD

**Discussion by masonlee93** [XTLS/Xray-core#2220] [Q&A]
目前得到确认的是，企业CN2并未受此影响，家庭163高优先也未受影响，受到影响的只有上海电信58.32.0.0/16段。
官方说法是监测到大量异常流量，以禁止家宽商用的名义对某些国内ip屏蔽。但实际上很小众的中转机场也被屏蔽。
鉴于目前中转机场全部用ss或ssr这些不安全的协议，是否可以猜想，国内实际上已经在部署监测或屏蔽境内加密协议流量，并在cn2线路进行试点。

**Comment by RPRX** [comment]
> 鉴于目前中转机场全部用ss或ssr这些不安全的协议，是否可以猜想，国内实际上已经在部署监测或屏蔽境内加密协议流量，并在cn2线路进行试点。

https://github.com/net4people/bbs/issues/254#issuecomment-1565308254
https://github.com/XTLS/Xray-core/pull/2173#issuecomment-1575617999

GFW 当然知道，不封不就是为了下套嘛，境内机房当然也知道，当没看见，不清退不就是为了最后再捞点钱嘛，明年后年可能就捞不了钱了

**Reply by nursery01** [reply]
你好不要臉，每次都把自己標記為答案

**Comment by lxsq** [comment]
杀PCDN被误伤了，PCDN群里可是一片哀嚎。

**Comment by eatbaba** [comment]
国内也有省墙？之前用流量中转服务，跑ss 2016版本+流加密。不到一个月连不上国内入口，表现为跨省ping不通，同一地区里面的服务器可以ping通。

然后换了IP，继续跑ss2016。然后过了两个月，经常掉线，tcpdump发现经常收到RST包。(连接国内入口服务器)。

连接旧Dst IP会有概率RST，开始以为是落地服务器问题，但是换了Dst IP就没有收到RST包。

收到RST包时，出国丢包率会上升。重新拨号换Src IP就正常。

现在又换了一个国内入口IP，继续ss 2016。

~感觉应该不是我祸害的，毕竟国内入口到国外出口是商家加密的~
